x-pack/plugins/apm/common/anomaly_detection.ts

```diff
@@ -4,9 +4,73 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 export interface ServiceAnomalyStats {
   transactionType?: string;
   anomalyScore?: number;
   actualValue?: number;
   jobId?: string;
 }
+
+export enum severity {
+  critical = 'critical',
+  major = 'major',
+  minor = 'minor',
+  warning = 'warning',
+}
+
+export function getSeverity(score?: number) {
```

Comment:
This function is very similar to some utility functions in the ML plugin which are already being shared.
If possible, it makes sense to me that you to use our functions just in case we decide that the anomaly scoring thresholds should change.
`getSeverityType` and `ANOMALY_SEVERITY` are available from `....../plugins/ml/public`

Reply:
Oh, that's great. I'll take a look. Thanks for jumping in @jgowdyelastic !